core/src/actions/properties.ts

```diff
@@ -50,30 +54,31 @@ export class PropertiesList extends AuthenticatedAction {
     > = [];
     const responseExamples: { [id: string]: string[] } = {};
 
-    for (const i in properties) {
-      const property = properties[i];
+    for (const property of properties) {
       const apiData = await property.apiData();
-
-      const examples = await ProfileProperty.findAll({
-        where: {
-          propertyId: property.id,
-          rawValue: { [Op.not]: null },
-        },
-        order: [["id", "asc"]],
-        limit: 5,
-      });
-      const exampleValues = examples.map((e) => e.rawValue);
-
       responseProperties.push(apiData);
-      responseExamples[property.id] = exampleValues;
+
+      if (includeExamples) {
```

Comment:
can probably do an N=1 Eager load fix later if you want.

Reply:
Can you share more about how that applies in this case?  We don't want to use the same 5 profiles for each Property, because they may have null values... so we search for 5 non-null ProfileProperties for each Property.

Reply:
I believe it's conceptaully similar to `User` that `hasMany` `Tools` like in this example.
https://sequelize.org/master/manual/eager-loading.html

In our case
In the model: `Property.hasMany(ProfileProperties)`

In this file:
```
const properties = await Property.findAll({
  include: {
    model: ProfileProperty,
    where: {
      rawValue: { [Op.not]: null },
    },
  }
});

for (const property of property) {
  const examples = property.profileProperties;
  // use that
}
```

I suppose the issue is the `limit` situation.
There is this `separate` thing.
```
const properties = await Property.findAll({
  include: {
    model: ProfileProperty,
    where: {
      rawValue: { [Op.not]: null },
    },
    order: [["id", "asc"]],
    limit: 5,
    separate: true,
  }
});

for (const property of property) {
  const examples = property.profileProperties;
  // use that
}
```
I would check to see if it's smarter than what we are doing manually here (or maybe it's the same).